src/docs/Components/Buttons/code.mdx

```diff
@@ -122,33 +122,42 @@ To manage display of your button, you have 2 availables classes :
 
 You can add an icon into any buttons you want.
 
-You should add 2 CSS Classes to your icon HTML tag :
+You can add 2 CSS Classes to your icon HTML tag :
 
 - Element class : `mc-button__icon`
 - Modifier element class : `mc-button__icon--[size]`
 
 ```html
 <a href="#" class="mc-button">
   <span class="mc-button__label">Label</span>
-  <svg class="mc-button__icon mc-button__icon--m">
+  <svg class="mc-button__icon mc-button__icon--l">
     <use xlink:href="#icon" />
   </svg>
 </a>
 ```
 
-<Preview path="button-icon-position" />
-
 #### Icon sizes
 
-As you see above, you need to add a modifier css classes to define the icon size.
+As you see above, you can add a modifier css classes to define the icon size.
 
 Available sizes are :
 
-- **S** : `mc-button__icon--s`
-- **M** : `mc-button__icon--m`
+- **M** : this is the default size so you don't need to add a modifier class
 - **L** : `mc-button__icon--l`
 - **XL** : `mc-button__icon--xl`
```

Comment:
We should tell users to use icon size X or Y in button sizes X or Y. (the actual SVG size)

Reply:
It is no longer necessary to specify it because the size of the icons is directly related to the size of the buttons now

Reply:
it is necessary to document the Icon size, not CSS wise but the svg file required for each button size.